ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -62,7 +71,30 @@ export class ImportCommand extends IronfishCommand {
     }
 
     const rescan = flags.rescan
-    const result = await client.importAccount({ account, rescan })
+
+    let result: RpcResponseEnded<ImportResponse>
+    try {
+      result = await client.importAccount({ account, rescan })
```

Comment:
Instead of trying and failing, can we use the `AccountImport.name` field and compare to existing accounts in the wallet? Then prompt for rename if there is a duplicate?

Reply:
Yeah that sounds good.